dfdaemon/downloader/dfget/dfget.go

```diff
@@ -44,6 +45,8 @@ type DFGetter struct {
 	notbs bool
 }
 
+var _ downloader.Interface = &DFGetter{}
```

Comment:
This cloud be written:

```go
var _ downloader.Interface = (*DFGetter)(nil)
```

Reply:
Yeah, has the same effect. 
I prefer to `&DFGetter{}` because it looks a little cleaner and is common in [kubernetes](https://github.com/kubernetes/kubernetes/blob/master/pkg/ssh/ssh.go#L126).